patches.json

```diff
@@ -42,64 +42,188 @@
   "MDVA-30131": {
     "magento/magento2-base": {
       "Fixes the issue with layered navigation, where the 'No' value for boolean type product attributes was not included in layered navigation if Elasticsearch was used as a search engine.": {
-        ">=2.3.4 <=2.3.5-p2 || 2.4.0": {
+        ">=2.3.4 <2.3.6 || 2.4.0": {
           "file": "os/MDVA-30131__fix_storing_boolean_values_for_custom_attributes_in_Elastic_Search__2.3.4.patch"
         }
       }
     }
   },
   "MDVA-27825": {
     "magento/magento2-base": {
-      "Fixes an issue where exporting of big amounts of data failed because of memory leak.": {
-        ">=2.3.0 <=2.3.5-p2 || 2.4.0": {
+      "Fixes the issue where exporting of big amounts of data failed because of memory leak.": {
+        ">=2.3.0 <2.4.1": {
           "file": "os/MDVA-27825__fix_memory_leak_prevent_customer_export__2.3.3.patch"
         }
       }
     }
   },
   "MDVA-29085": {
     "magento/magento2-b2b-base": {
-      "Fixes a B2B issue where no required new company emails are sent out if a company is created by API.": {
-        ">=1.1.0 <=1.1.5-p1": {
+      "Fixes the B2B issue where no required new company emails are sent out if a company is created by API.": {
+        ">=1.1.0 <1.2.2": {
```

Comment:
Looks like this patch is not compatible with Magento 2.4.0 (B2B 1.2.0)

Reply:
fixed